### PR TITLE
Update client.py

### PR DESCRIPTION
**Pull request by Roushangopal**
Improved code readability and removed two consecutive single line comment with one multiline comment.

### CODE REVIEW

**Comment by Roushangopal** [comment]
I have understood your base code will be implementing some cool features in the future soon

**Comment by Roushangopal** [comment]
I have understood your base code will be implementing some cool features in the future soon

**Comment by cclauss** [comment]
You might run this code through the Python Software Foundation’s code formater to see if it agrees with your changes... https://github.com/psf/black

File: trough/client.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def __init__(self, rethinkdb_trough_db_url, promotion_interval=None):
         '''
         parsed = doublethink.parse_rethinkdb_url(rethinkdb_trough_db_url)
         self.rr = doublethink.Rethinker(
-                servers=parsed.hosts, db=parsed.database)
+                servers = parsed.hosts, db = parsed.database)
```
Comment (cclauss) [inline trough/client.py:62]: From [PEP8](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements):
```

Don't use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:

# Correct:
def complex(real, imag=0.0):
    return magic(r=real, i=imag)
# Wrong:
def complex(real, imag = 0.0):
    return magic(r = real, i = imag)
```

File: trough/client.py [hunk 2]
```diff
@@ -175,7 +177,7 @@ def read_urls_for_regex(self, regex):
         `{segment: url}`
         '''
         d = {}
-        reql = self.rr.table('services', read_mode='outdated')\
+        reql = self.rr.table('services', read_mode = 'outdated')\
```
Comment (cclauss) [inline trough/client.py:180]: The cleanup here should be to replace the backslashes with () as discussed in PEP8.

**Pull request closed without merging**